KeyboardHeightConstraint.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  KeyboardHeightConstraint.swift
+//  FuntastyKit-iOS
+//
+//  Created by Matěj Jirásek on 07/02/2018.
+//  Copyright © 2018 The Funtasty. All rights reserved.
+//
+
+import UIKit
+
+final class KeyboardHeightConstraint: NSLayoutConstraint {
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        NotificationCenter.default.addObserver(self, selector: #selector(keyboardWillChange(_:)), name: .UIKeyboardWillChangeFrame, object: nil)
```

Comment:
Shouldn't we also add `deinit` with observer removal?

Reply:
Since iOS 9 it works automatically in this case, when the constraint is deallocated the observer is removed. It was tested.